### ISSUE TITLE
Load design 

### ISSUE THREAD

**Issue by thewebprojects** [unlayer/react-email-editor#114]
Hi. 

I can load the editor, save to my database, etc. But I cant load an initial, very simple template.

Loading a template seems real easy, but i must be missing something. It doesn't load. The editor doesn't even load. I created a template within my unlayer account, grabbed the json, minified it, and used the load function.

`unlayer.loadDesign('my json code here')`

Does the json need to be escaped or formatted in some way?

Please advise

**Comment by thewebprojects** [comment]
And I'm back... I discovered the correct way to load json.  I had to ditch those single quotes.

`unlayer.loadDesign({my json code here})`

**Comment by naivedeveloper95** [comment]
I can load the template now, but I am unable to upload an image in an existing template. The editor responded with this message in the console.

**contents:10 cannot be updated because it is already deleted.**

But I can add images from the Images Block (This is a premium feature).

I need to fix this on priority. Please help.

**Comment by thewebprojects** [comment]
@KevinMull, No. I didn't. In an effort to be helpful here's a [related article](https://docs.microsoft.com/en-us/sql/relational-databases/json/store-json-documents-in-sql-tables?view=sql-server-ver16). Good luck.
